libraries/Microsoft.Bot.Builder.AI.QnA/QnAMakerExtensions.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Text.RegularExpressions;
+using Microsoft.Extensions.Configuration;
+
+namespace Microsoft.Bot.Builder.AI.QnA
+{
+    /// <summary>
+    /// Extension methods for QnA.
+    /// </summary>
+    public static class QnAMakerExtensions
```

Comment:
How is this related to the issue associated with this PR?

Reply:
I have splitted this PR into two new PRs.